shared/actions/chat/index.js

```diff
@@ -563,12 +573,14 @@ function _unboxedToMessage(
 
     return {
       author: yourName,
+      channelMention: 'None',
```

Comment:
are these valid initial values?

Reply:
yeah, I think so

Reply:
Why wouldn't you use the value generated from AVDL? It already has a "NONE" value for this.

Reply:
It looks like we convert from the AVDL type (a union of numbers) to our own string union: see `_parseChannelMention`. Maybe for easier logging?

Reply:
Seem unnecessary and probably bug prone, if it is better for logging just convert it at that time. Don't care about it if it is a big change, but just seemed odd when I saw it.

Reply:
Might be a big change... @chrisnojima or @MarcoPolo is there a reason for this conversion I'm missing?